### DISCUSSION TITLE
Collection of 248 Base16 Themes for mini.base16

### DISCUSSION THREAD

**Discussion by wroyca** [nvim-mini/mini.nvim#370] [Ideas]
Here's a base16 themes compilation that have been adapted to mini.base16. These themes were initially ported to Lua by someone else, and my role was only to port them to mini.base16 - unfortunately, I cannot recall who performed the Lua port initially, so please feel free to provide any information if you happen to know!

<details>
 <summary>Here are a few screenshots for reference.</summary>

![Screenshot from 2023-06-15 14-20-53](https://github.com/echasnovski/mini.nvim/assets/91024200/d3bc6196-24fe-4cbf-a0de-9f476b1e8d8e)
![Screenshot from 2023-06-15 14-20-55](https://github.com/echasnovski/mini.nvim/assets/91024200/28051b74-2a07-4b0b-ad30-958b7207846e)
![Screenshot from 2023-06-15 14-20-56](https://github.com/echasnovski/mini.nvim/assets/91024200/944a63b9-8db4-4ec8-af88-95e0cc0e9a67)
![Screenshot from 2023-06-15 14-20-57](https://github.com/echasnovski/mini.nvim/assets/91024200/13df846a-69dc-49d0-834a-2cf714731ad2)
![Screenshot from 2023-06-15 14-20-58](https://github.com/echasnovski/mini.nvim/assets/91024200/312885b6-438f-4126-8958-85f6f0951804)
![Screenshot from 2023-06-15 14-20-59](https://github.com/echasnovski/mini.nvim/assets/91024200/6bb443d9-8b99-4be1-b575-eb698d1c7b47)
![Screenshot from 2023-06-15 14-21-00](https://github.com/echasnovski/mini.nvim/assets/91024200/4ccf29ad-99dd-459c-8316-35d17e16ba89)
![Screenshot from 2023-06-15 14-21-00-1](https://github.com/echasnovski/mini.nvim/assets/91024200/a780b377-b423-4125-902a-33730d6da6a4)
![Screenshot from 2023-06-15 14-21-01](https://github.com/echasnovski/mini.nvim/assets/91024200/afe6d590-a591-4126-badd-083c8182905e)
![Screenshot from 2023-06-15 14-21-02](https://github.com/echasnovski/mini.nvim/assets/91024200/9887b444-9c8b-4739-bc41-ec6b1228b655)
![Screenshot from 2023-06-15 14-21-03](https://github.com/echasnovski/mini.nvim/assets/91024200/c7c1f9cf-0f25-458a-bf19-f01f2dea811b)
![Screenshot from 2023-06-15 14-21-04](https://github.com/echasnovski/mini.nvim/assets/91024200/b6c2f66d-0b36-45c8-b917-07f054afe5e0)
![Screenshot from 2023-06-15 14-21-04-1](https://github.com/echasnovski/mini.nvim/assets/91024200/66c7f2d3-5282-4291-a0d4-6525d480f240)
![Screenshot from 2023-06-15 14-21-05](https://github.com/echasnovski/mini.nvim/assets/91024200/39c4d23b-0eaa-45a9-affd-b578961d192b)
![Screenshot from 2023-06-15 14-21-06](https://github.com/echasnovski/mini.nvim/assets/91024200/f5b4f0fe-24dd-4594-a2e6-b5a533cca18a)
![Screenshot from 2023-06-15 14-21-07](https://github.com/echasnovski/mini.nvim/assets/91024200/357f9ce6-f260-43ef-96e2-069fb4886b09)
![Screenshot from 2023-06-15 14-21-08](https://github.com/echasnovski/mini.nvim/assets/91024200/17f142a8-aa43-460b-a49c-1343ffb133d3)
![Screenshot from 2023-06-15 14-21-09](https://github.com/echasnovski/mini.nvim/assets/91024200/6a05a889-6eaf-464c-a070-6379389a617b)
![Screenshot from 2023-06-15 14-21-10](https://github.com/echasnovski/mini.nvim/assets/91024200/7511c1c2-1dc5-4022-be2b-b2854102abf9)
![Screenshot from 2023-06-15 14-21-11](https://github.com/echasnovski/mini.nvim/assets/91024200/9da9f7f9-9a8a-4872-9373-922de61abea4)
![Screenshot from 2023-06-15 14-21-11-1](https://github.com/echasnovski/mini.nvim/assets/91024200/b6efca7e-e4f1-4ad6-a45d-a16b9ded327b)
![Screenshot from 2023-06-15 14-21-12](https://github.com/echasnovski/mini.nvim/assets/91024200/dcfb7f40-3ecd-4810-8506-a954dff674e1)
![Screenshot from 2023-06-15 14-21-13](https://github.com/echasnovski/mini.nvim/assets/91024200/1505bbcf-21c4-4642-8134-93e63a493018)
![Screenshot from 2023-06-15 14-21-14](https://github.com/echasnovski/mini.nvim/assets/91024200/6f0706b1-ec41-4335-a8e3-6295ba02448c)
![Screenshot from 2023-06-15 14-21-15](https://github.com/echasnovski/mini.nvim/assets/91024200/a1a6ce38-06fb-4811-9287-c8c6d9724754)
![Screenshot from 2023-06-15 14-21-16](https://github.com/echasnovski/mini.nvim/assets/91024200/b392f349-4144-4c8c-af43-2b92cec314d6)
![Screenshot from 2023-06-15 14-21-17](https://github.com/echasnovski/mini.nvim/assets/91024200/6fdf8554-1acf-45fe-a164-5509ffc0fc88)
![Screenshot from 2023-06-15 14-21-18](https://github.com/echasnovski/mini.nvim/assets/91024200/1afec817-f5d2-438a-8b5d-1186868fbbcb)
![Screenshot from 2023-06-15 14-21-18-1](https://github.com/echasnovski/mini.nvim/assets/91024200/ce1094ba-947f-47f2-912e-6a57951ac4aa)
![Screenshot from 2023-06-15 14-21-19](https://github.com/echasnovski/mini.nvim/assets/91024200/0dcc82cf-a5e1-4224-95a5-052d7aacc300)
![Screenshot from 2023-06-15 14-21-20](https://github.com/echasnovski/mini.nvim/assets/91024200/d52cfca1-36d7-4aea-b1b9-57437d135bb9)
![Screenshot from 2023-06-15 14-21-21](https://github.com/echasnovski/mini.nvim/assets/91024200/e77fd6ac-95e7-46d4-b332-b59cec031ae5)
![Screenshot from 2023-06-15 14-21-22](https://github.com/echasnovski/mini.nvim/assets/91024200/e812ab57-f881-41f3-a330-5407cc66c78d)
![Screenshot from 2023-06-15 14-21-22-1](https://github.com/echasnovski/mini.nvim/assets/91024200/a1cd3d5a-db8e-4d77-b14d-ddce2355b76a)
![Screenshot from 2023-06-15 14-21-23](https://github.com/echasnovski/mini.nvim/assets/91024200/4200cf32-ddc6-4586-bf59-b0d6b3bb0977)
![Screenshot from 2023-06-15 14-21-24](https://github.com/echasnovski/mini.nvim/assets/91024200/caa5612b-c96e-4295-b646-773a758080a2)
![Screenshot from 2023-06-15 14-21-25](https://github.com/echasnovski/mini.nvim/assets/91024200/66ab31fc-8c23-4b04-afd5-772c1b97d8d3)
![Screenshot from 2023-06-15 14-21-26](https://github.com/echasnovski/mini.nvim/assets/91024200/5ace737f-f25f-4bc4-ae3e-195eed303864)
![Screenshot from 2023-06-15 14-21-27](https://github.com/echasnovski/mini.nvim/assets/91024200/b8a91eaf-94b2-498b-a828-c250cf73f757)
![Screenshot from 2023-06-15 14-21-27-1](https://github.com/echasnovski/mini.nvim/assets/91024200/5b51bf2c-05d7-4d4d-bb6e-475bdddde7f0)
![Screenshot from 2023-06-15 14-21-28](https://github.com/echasnovski/mini.nvim/assets/91024200/0a7b678b-0541-43c7-99bd-8f8bf81190b2)
![Screenshot from 2023-06-15 14-21-29](https://github.com/echasnovski/mini.nvim/assets/91024200/0007db1c-72b4-417d-b312-6c3d3627d46e)
![Screenshot from 2023-06-15 14-21-30](https://github.com/echasnovski/mini.nvim/assets/91024200/b20d5dfc-444b-4356-b234-54f93b78100c)
![Screenshot from 2023-06-15 14-21-31](https://github.com/echasnovski/mini.nvim/assets/91024200/9794bc8e-9e14-451f-a1d8-9e32700b831b)
![Screenshot from 2023-06-15 14-21-32](https://github.com/echasnovski/mini.nvim/assets/91024200/7f8db5b1-4bb0-4ee7-9c89-ef86a80715af)
![Screenshot from 2023-06-15 14-21-32-1](https://github.com/echasnovski/mini.nvim/assets/91024200/8c77a904-c17c-41b6-aa80-2f9135f6288c)
![Screenshot from 2023-06-15 14-21-33](https://github.com/echasnovski/mini.nvim/assets/91024200/0db3f603-adc3-4d36-91cb-b6498567ed3b)
![Screenshot from 2023-06-15 14-21-34](https://github.com/echasnovski/mini.nvim/assets/91024200/3141eff6-03f6-46db-9e81-a5ec75ed4372)
![Screenshot from 2023-06-15 14-21-35](https://github.com/echasnovski/mini.nvim/assets/91024200/22c3a34b-1e3c-4927-b60a-b20a90c96d48)
![Screenshot from 2023-06-15 14-21-36](https://github.com/echasnovski/mini.nvim/assets/91024200/e9211006-3b69-4a3c-9e93-97aa92d1fa65)
![Screenshot from 2023-06-15 14-21-37](https://github.com/echasnovski/mini.nvim/assets/91024200/7458ae6d-533b-4b0f-be5e-ea65f74d493f)
![Screenshot from 2023-06-15 14-21-37-1](https://github.com/echasnovski/mini.nvim/assets/91024200/f18320d4-28b7-4ec1-bd9f-014cbdcb92b3)
![Screenshot from 2023-06-15 14-21-38](https://github.com/echasnovski/mini.nvim/assets/91024200/c2407a38-82d6-4f44-97c0-dd48f3c8eab3)
![Screenshot from 2023-06-15 14-21-39](https://github.com/echasnovski/mini.nvim/assets/91024200/29fec961-5af1-4208-88b7-c49981a5e902)
![Screenshot from 2023-06-15 14-21-40](https://github.com/echasnovski/mini.nvim/assets/91024200/5808e441-d2ad-4a7e-9d16-0b68359840da)
![Screenshot from 2023-06-15 14-21-41](https://github.com/echasnovski/mini.nvim/assets/91024200/565010d6-9f61-4d4c-9ebe-d6ab8d81e0cd)
![Screenshot from 2023-06-15 14-21-42](https://github.com/echasnovski/mini.nvim/assets/91024200/ca416de3-6f1e-4601-a1be-c350552e5723)
![Screenshot from 2023-06-15 14-21-43](https://github.com/echasnovski/mini.nvim/assets/91024200/37bb5413-cbea-4905-ad3a-aa652e74aa5c)
![Screenshot from 2023-06-15 14-21-43-1](https://github.com/echasnovski/mini.nvim/assets/91024200/02345c6e-d0fd-4b91-bebf-d4c1e0c6c43d)
![Screenshot from 2023-06-15 14-21-44](https://github.com/echasnovski/mini.nvim/assets/91024200/d3d03de5-53bb-4ff6-bc45-3927aae46e29)
![Screenshot from 2023-06-15 14-21-45](https://github.com/echasnovski/mini.nvim/assets/91024200/bd7bc795-4804-4cb6-9c90-b3a6e631db9b)
![Screenshot from 2023-06-15 14-21-46](https://github.com/echasnovski/mini.nvim/assets/91024200/ec721e5a-42a4-40f2-bba4-cd9b9cfe6aa3)
![Screenshot from 2023-06-15 14-21-47](https://github.com/echasnovski/mini.nvim/assets/91024200/2c40bbeb-e000-48f9-a6db-3970e5dbed8c)
![Screenshot from 2023-06-15 14-21-47-1](https://github.com/echasnovski/mini.nvim/assets/91024200/3d06cbf8-a77a-4148-b1dd-d98607707793)
![Screenshot from 2023-06-15 14-21-48](https://github.com/echasnovski/mini.nvim/assets/91024200/9af898ce-a5d2-49fa-9913-a08430478704)
![Screenshot from 2023-06-15 14-21-49](https://github.com/echasnovski/mini.nvim/assets/91024200/86e5d552-0e96-4032-8e86-4f6f14c03953)
![Screenshot from 2023-06-15 14-21-50](https://github.com/echasnovski/mini.nvim/assets/91024200/63a7a009-1462-419d-9489-e1088b08f582)
![Screenshot from 2023-06-15 14-21-51](https://github.com/echasnovski/mini.nvim/assets/91024200/ad7c4b67-1cb1-4fff-a08c-0ea4263ab200)
![Screenshot from 2023-06-15 14-21-51-1](https://github.com/echasnovski/mini.nvim/assets/91024200/76bda7cd-b8de-4754-9e49-3493cdb86289)
![Screenshot from 2023-06-15 14-21-52](https://github.com/echasnovski/mini.nvim/assets/91024200/4eae9bb8-b216-481a-8258-4d25dae75248)
![Screenshot from 2023-06-15 14-21-53](https://github.com/echasnovski/mini.nvim/assets/91024200/4ed634ab-a96a-43eb-bb4b-0ff4d8182d4a)
![Screenshot from 2023-06-15 14-21-54](https://github.com/echasnovski/mini.nvim/assets/91024200/504eabe7-eafa-42ad-9c26-3ff1b08027e2)
![Screenshot from 2023-06-15 14-21-55](https://github.com/echasnovski/mini.nvim/assets/91024200/d4b9ab35-e262-4a3c-b6a4-bc9b96252059)
![Screenshot from 2023-06-15 14-21-56](https://github.com/echasnovski/mini.nvim/assets/91024200/25f5a9e4-8886-4a00-a88b-a7b0d8049d66)
![Screenshot from 2023-06-15 14-21-56-1](https://github.com/echasnovski/mini.nvim/assets/91024200/ea9d921c-251f-4e69-b736-a5d39eed5b99)
![Screenshot from 2023-06-15 14-21-57](https://github.com/echasnovski/mini.nvim/assets/91024200/a845c852-bd18-4f54-a883-6d3d0bbf04ef)
![Screenshot from 2023-06-15 14-21-58](https://github.com/echasnovski/mini.nvim/assets/91024200/d43ad38c-bf4d-41e0-8ab2-97ec1b115087)
![Screenshot from 2023-06-15 14-21-59](https://github.com/echasnovski/mini.nvim/assets/91024200/44d59099-448e-4cda-a8e6-179de331213e)
![Screenshot from 2023-06-15 14-22-00](https://github.com/echasnovski/mini.nvim/assets/91024200/d8302143-775a-4386-bb78-8988d382c346)
![Screenshot from 2023-06-15 14-22-00-1](https://github.com/echasnovski/mini.nvim/assets/91024200/362961f3-073a-44b7-bae1-b822e9bcfea0)
![Screenshot from 2023-06-15 14-22-01](https://github.com/echasnovski/mini.nvim/assets/91024200/99244729-9ac4-41ce-b819-28181426211d)
![Screenshot from 2023-06-15 14-22-02](https://github.com/echasnovski/mini.nvim/assets/91024200/035f9f39-38a1-4f3a-8c84-1729f08113fb)
![Screenshot from 2023-06-15 14-22-02-1](https://github.com/echasnovski/mini.nvim/assets/91024200/7afdafd0-bcc6-4867-a1c5-a3beb0ec6c1a)
![Screenshot from 2023-06-15 14-22-03](https://github.com/echasnovski/mini.nvim/assets/91024200/03a7c86e-70dd-48b8-9404-7e0dc592cb55)
![Screenshot from 2023-06-15 14-22-04](https://github.com/echasnovski/mini.nvim/assets/91024200/f28b5d8c-0e98-400b-b3c2-5e830934e3c1)
![Screenshot from 2023-06-15 14-22-04-1](https://github.com/echasnovski/mini.nvim/assets/91024200/af358b6e-bf9b-4b16-b8db-67b46c3cea09)
![Screenshot from 2023-06-15 14-22-05](https://github.com/echasnovski/mini.nvim/assets/91024200/589cd6af-0671-4a3a-9a97-b1798be8a1bf)
![Screenshot from 2023-06-15 14-22-06](https://github.com/echasnovski/mini.nvim/assets/91024200/fcb8cc10-41cb-448c-98a7-1c5a805916b4)
![Screenshot from 2023-06-15 14-22-07](https://github.com/echasnovski/mini.nvim/assets/91024200/25b71528-7a2d-48a7-a445-2bac6f09728f)
![Screenshot from 2023-06-15 14-22-07-1](https://github.com/echasnovski/mini.nvim/assets/91024200/a1ce6c08-413a-4d5c-83d3-0e9c3eb6c2b4)
![Screenshot from 2023-06-15 14-22-08](https://github.com/echasnovski/mini.nvim/assets/91024200/96372813-9611-48ff-aecc-160426443add)
![Screenshot from 2023-06-15 14-22-09](https://github.com/echasnovski/mini.nvim/assets/91024200/23c34f8c-3574-476c-891e-08271d62bc48)
![Screenshot from 2023-06-15 14-22-09-1](https://github.com/echasnovski/mini.nvim/assets/91024200/e4642c58-560e-495f-ba13-d3a408612410)
![Screenshot from 2023-06-15 14-22-10](https://github.com/echasnovski/mini.nvim/assets/91024200/aef8ee37-b6a8-4c7f-a07a-dcb76928bb01)
![Screenshot from 2023-06-15 14-22-10-1](https://github.com/echasnovski/mini.nvim/assets/91024200/361e498d-1dc0-4555-987d-361c7de01393)
![Screenshot from 2023-06-15 14-22-11](https://github.com/echasnovski/mini.nvim/assets/91024200/8d22fdcd-01e6-4de3-99af-9e7708f46eb9)
![Screenshot from 2023-06-15 14-22-11-1](https://github.com/echasnovski/mini.nvim/assets/91024200/9a749998-7ce8-470c-b1bd-32f11f6de919)

</details>

[base16-.zip](https://github.com/echasnovski/mini.nvim/files/11761251/base16-.zip)

**Comment by echasnovski** [comment]
This is a very nice zip file you've got there. Thanks for sharing!

**Comment by wroyca** [comment]
Hey @echasnovski, 

Are you open to the idea of a (gradual) pull request to bundle them with mini.base16? I'm currently in the process of slowly setting it up, merging the dark and light variants, addressing any highlight issues that may arise, and incorporating some adjustments based on feedback. Given that mini.base16 already includes minicyan.lua and minischeme.lua, Perhaps you might be interested in this proposal

**Reply by echasnovski** [reply]
Sorry, no, I don't think so. The point of 'mini.base16' is to serve as color scheme generator. Both 'minischeme' and 'minicyan' originally are there so that 'mini.nvim' can come with an already configured color scheme.

When you finish polishing, we can discuss how to mention your collection. Either in 'mini.base16' README or in wiki, or somewhere else.

**Comment by wroyca** [comment]
Atelier collection, now with proper dark/light background switch and cterm support. cc @echasnovski :) 

![image](https://github.com/echasnovski/mini.nvim/assets/91024200/2c0970c0-f75f-47f7-bee5-2bf2b0df9d93)

[readonly_base16-atelier.zip](https://github.com/echasnovski/mini.nvim/files/12823228/readonly_base16-atelier.zip)
